config/default/kustomization.yaml

```diff
@@ -29,7 +29,6 @@ patchesStrategicMerge:
   # Protect the /metrics endpoint by putting it behind auth.
```

Comment:
remove these comments as well

Reply:
Those should be auto-generated, let me change to comment this line, instead of delete it

Reply:
Those should be auto-generated, let me change to comment this line, instead of delete it

Reply:
thanks, that sounds good